Files/Interacts/Interaction.cs

```diff
@@ -888,7 +889,7 @@ public async Task PasteItems(DataPackageView packageView, string destinationPath
             {
                 if (item.IsOfType(StorageItemTypes.Folder))
                 {
-                    if (destinationPath.Contains(item.Path, StringComparison.OrdinalIgnoreCase))
```

Comment:
Instead of adding a full class and removing this line, would it work to change `Contains` to `Equals`?

Reply:
"D:/Some Folder" copy to "D:/Some Folder/Some Folder another/...". Second folder is subfolder for first folder. But paths are not equals

Reply:
Ok, thanks for explaining.